stack-guideline.md

```diff
@@ -0,0 +1,65 @@
+# Abtion stack
+###### Or Ab-*stack*-tion!  🤦‍♂️
+![abtion image](https://media.giphy.com/media/ZVik7pBtu9dNS/giphy.gif)
+
+
+In Abtion we strive for the most efficient and easiest workflow.
```

Comment:
```suggestion
In Abtion we strive for the easiest and most efficient workflow.
```
I am fairly sure this is correct... can't tell you why 🤔

Reply:
It is wrong to say most easiest.
This way 'easiest' is definitely not associated with 'most'.

For instance, in 'most efficient and clever', most is associated with both words, which makes sense.